File: Programing_Basics_with_JS/examPreparation/programmingBasicsOnlineExam20And21April2019/03.EasterTrip.js
function easterTrip(input) {
    let destination = input[0];
    let date = input[1];
    let nightsCount = Number(input[2]);
    
    let nightsPrice = 0;

    switch(destination) {
        case 'France':
            switch(date) {
                case '21-23': nightsPrice = 30; break;
                case '24-27': nightsPrice = 35; break;
                case '28-31': nightsPrice = 40; break;
            } break;
        
         case 'Italy':
            switch(date) {
                case '21-23': nightsPrice = 28; break;
                case '24-27': nightsPrice = 32; break;
                case '28-31': nightsPrice = 39; break;
            } break;

         case 'Germany':
            switch(date) {
                case '21-23': nightsPrice = 32; break;
                case '24-27': nightsPrice = 37; break;
                case '28-31': nightsPrice = 43; break;
            } break;
    }

    let allCost = nightsCount * nightsPrice;
    console.log(`Easter trip to ${destination} : ${allCost.toFixed(2)} leva.`);
}

easterTrip(["Germany",
"24-27",
"5"])

easterTrip(["Italy",
"21-23",
"7"])

easterTrip(["France",
"28-31",
"8"])

File: Programming_Basics_with_Python/programming_basics_online_exam_20_and_21_april_2019/03._easter_trip.py
trip_destination = input()
trip_date = input()
nights_numbers = int(input())

total_cost = 0
if trip_destination == 'France':
    if trip_date == '21-23':
        night_price = 30
        total_cost = night_price * nights_numbers
    elif trip_date == '24-27':
        night_price = 35
        total_cost = night_price * nights_numbers
    elif trip_date == '28-31':
        night_price = 40
        total_cost = night_price * nights_numbers
elif trip_destination == 'Italy':
    if trip_date == '21-23':
        night_price = 28
        total_cost = night_price * nights_numbers
    elif trip_date == '24-27':
        night_price = 32
        total_cost = night_price * nights_numbers
    elif trip_date == '28-31':
        night_price = 39
        total_cost = night_price * nights_numbers
elif trip_destination == 'Germany':
    if trip_date == '21-23':
        night_price = 32
        total_cost = night_price * nights_numbers
    elif trip_date == '24-27':
        night_price = 37
        total_cost = night_price * nights_numbers
    elif trip_date == '28-31':
        night_price = 43
        total_cost = night_price * nights_numbers

print(f'Easter trip to {trip_destination} : {total_cost:.2f} leva.')